__tests__/pages/home.test.js

```diff
@@ -4,13 +4,19 @@
 import { render, screen } from "@testing-library/react";
 import Home from "../../pages/home";
 import { homePageData } from "../../__mocks__/mockStore";
+import { experimentsData } from "../../__mocks__/mockStore";
 
 describe("Home", () => {
   it("renders without crashing", () => {
-    render(<Home pageData={homePageData.data.sCLabsPageByPath} />);
+    render(
+      <Home
+        pageData={homePageData.data.scLabsPagev1ByPath}
+        experimentsData={experimentsData.data.scLabsPagev1List.items[2]}
+      />
+    );
     expect(
       screen.getByRole("heading", {
-        name: "Service Canada Labs(FR)",
+        name: "Laboratoires de Service Canada",
```

Comment:
Should this go in a global constant for reusability? Just a thought.

Reply:
Good point! Making a TODO